runtime/Cpp/runtime/src/atn/LexerActionType.h

```diff
@@ -14,7 +16,7 @@ namespace atn {
   /// @author Sam Harwell
   /// @since 4.2
   /// </summary>
-  enum class ANTLR4CPP_PUBLIC LexerActionType : size_t {
+  enum class LexerActionType : size_t {
```

Comment:
Aha, didn't think of that (maybe I got just confused by `enum *class*`. OK then.